proposals/lip-0045.md

```diff
@@ -1405,6 +1405,13 @@ def getMessageFeeTokenID(chainID: ChainID) -> TokenID:
     return channel(chainID).messageFeeTokenID
 ```
 
```

Comment:
Comment about the old `getMessageFeeTokenID` function:

In case the arg `chainID = OWN_CHAIN_ID` and `OWN_CHAIN_ID` is a sidechain, I guess the intended behavior is to output some error "input chain is own chain", but according to current specs it would just return the messageFeeTokenID used in the channel with mainchain.

Reply:
Yes, which is probably at least misleading. We should probably just return an error as you suggest.